required/latex-lab/config-math-luatex.lua

```diff
@@ -5,12 +5,6 @@ checkengines = {"luatex"}
 checksearch  = true
 testfiledir  = "testfiles-math-luatex"
 
-checkdeps =
-  {
-    maindir .. "/base",
-    maindir .. "/required/amsmath"
-  }
-
```

Comment:
not quite sure I understand why this means it is using the tools dir as a dependency.

Reply:
the main build.lua sets checkdeps to a larger set (which includes tools).

Reply:
@FrankMittelbach  What @u-fischer  said - we had a config-dependent override which didn't really make sense

Reply:
well I'm not sure it makes sense to have this hierachical. Seems to me it is better to set dependencies always for the test dir you work with, but fine, something to remember.

Reply:
We have a standard set of deps for the entire repo - the main reason for altering is where you'd get a circular ref